lib/node_modules/@stdlib/math/base/complex/signum/lib/index.js

```diff
@@ -0,0 +1,57 @@
+/**
+* @license Apache-2.0
+*
+* Copyright (c) 2018 The Stdlib Authors.
+*
+* Licensed under the Apache License, Version 2.0 (the "License");
+* you may not use this file except in compliance with the License.
+* You may obtain a copy of the License at
+*
+*    http://www.apache.org/licenses/LICENSE-2.0
+*
+* Unless required by applicable law or agreed to in writing, software
+* distributed under the License is distributed on an "AS IS" BASIS,
+* WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+* See the License for the specific language governing permissions and
+* limitations under the License.
+*/
+
+'use strict';
+
+/**
+* Find the signum function of a complex number.
```

Comment:
s/Find/Evaluate/

In pkg `index.js` files, `package.json` description, and top-level README description, we use the imperative voice; e.g., "evaluate" rather than "evaluates", describing what a package **will** do. Everywhere else, we use the active voice; e.g., "evaluates", describing what a function, block of code, etc, is actively doing.